src/common/copilot/dataverseMetadata.ts

```diff
@@ -64,9 +91,51 @@ function getAttributesFromResponse(jsonResponse: any): Attribute[] {
     return [];
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function getFormXMLFromResponse(jsonResponse: any): string {
+    if (jsonResponse.value[0].formxml) {
+        return jsonResponse.value[0].formxml;
+    }
+
+    return '';
+}
+
+function parseXML(formXml: string) {
+    const parser = new DOMParser();
+    const xmlDoc = parser.parseFromString(formXml, "text/xml");
+
+    // Get all 'row' elements
+    const rows = xmlDoc.getElementsByTagName('row');
+
+    const result = [];
+
+    // Iterate over all 'row' elements
+    for (let i = 0; i < rows.length; i++) {
```

Comment:
NIT: can we use forEach loop?

Reply:
'forEach' does not exist on type 'HTMLCollectionOf<Element>'

Reply:
You can use Array.from and then forEach